### DISCUSSION TITLE
Edit server type name

### DISCUSSION THREAD

**Discussion by Distractic** [Minestom/Minestom#1660] [Help] [answered]
In the F3 information, the server type is "Minestom"

![image](https://user-images.githubusercontent.com/46402441/210888195-b62b1641-2cd1-44c6-9a43-5703cb1a4893.png)

Is the value changeable ?

**Comment by Tofaa2** [comment]
You can change the brand name of the server by using MinecraftServer#setBrandName()

**Comment by GoldenStack** [comment, marked as answer]
MinecraftServer.setBrandName is what controls that:
https://javadoc.minestom.net/net/minestom/server/MinecraftServer.html#setBrandName(java.lang.String)
